diff --git a/build/windows/msi/i18n/scribe.de-de.wxl b/build/windows/msi/i18n/scribe.de-de.wxl
index ca92126..6b215dd 100644
--- a/build/windows/msi/i18n/scribe.de-de.wxl
+++ b/build/windows/msi/i18n/scribe.de-de.wxl
@@ -2,9 +2,9 @@
 <WixLocalization Culture="de-de" Codepage="1252" xmlns="http://schemas.microsoft.com/wix/2006/localization">
   <String Id="ProductLanguage">1031</String>
   <String Id="ProductName">@@PRODUCT_NAME@@</String>
-  <String Id="ProductHelpLink">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlInfoAbout">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlUpdateInfo">https://github.com/VSCodium/vscodium</String>
+  <String Id="ProductHelpLink">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlInfoAbout">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlUpdateInfo">https://github.com/BiblioNexus-Foundation/scribe</String>
   <String Id="PackageDescription">Paket mit @@PRODUCT_NAME@@</String>
   <String Id="FeatureStartMenuShortcutTitle">Startmenü-Verknüpfung</String>
   <String Id="FeatureStartMenuShortcutDescription">Startmenü-Symbol erstellen.</String>
diff --git a/build/windows/msi/i18n/scribe.en-us.wxl b/build/windows/msi/i18n/scribe.en-us.wxl
index 24303fa..7155de0 100644
--- a/build/windows/msi/i18n/scribe.en-us.wxl
+++ b/build/windows/msi/i18n/scribe.en-us.wxl
@@ -2,9 +2,9 @@
 <WixLocalization Culture="en-us" Codepage="1252" xmlns="http://schemas.microsoft.com/wix/2006/localization">
   <String Id="ProductLanguage">1033</String>
   <String Id="ProductName">@@PRODUCT_NAME@@</String>
-  <String Id="ProductHelpLink">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlInfoAbout">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlUpdateInfo">https://github.com/VSCodium/vscodium</String>
+  <String Id="ProductHelpLink">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlInfoAbout">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlUpdateInfo">https://github.com/BiblioNexus-Foundation/scribe</String>
   <String Id="PackageDescription">Package with @@PRODUCT_NAME@@</String>
   <String Id="FeatureStartMenuShortcutTitle">Start Menu shortcut</String>
   <String Id="FeatureStartMenuShortcutDescription">Create a start menu icon.</String>
diff --git a/build/windows/msi/i18n/scribe.es-es.wxl b/build/windows/msi/i18n/scribe.es-es.wxl
index af84628..75a792f 100644
--- a/build/windows/msi/i18n/scribe.es-es.wxl
+++ b/build/windows/msi/i18n/scribe.es-es.wxl
@@ -2,9 +2,9 @@
 <WixLocalization Culture="es-es" Codepage="1252" xmlns="http://schemas.microsoft.com/wix/2006/localization">
   <String Id="ProductLanguage">3082</String>
   <String Id="ProductName">@@PRODUCT_NAME@@</String>
-  <String Id="ProductHelpLink">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlInfoAbout">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlUpdateInfo">https://github.com/VSCodium/vscodium</String>
+  <String Id="ProductHelpLink">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlInfoAbout">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlUpdateInfo">https://github.com/BiblioNexus-Foundation/scribe</String>
   <String Id="PackageDescription">Package with @@PRODUCT_NAME@@</String>
   <String Id="FeatureStartMenuShortcutTitle">Start Menu shortcut</String>
   <String Id="FeatureStartMenuShortcutDescription">Create a start menu icon.</String>
diff --git a/build/windows/msi/i18n/scribe.fr-fr.wxl b/build/windows/msi/i18n/scribe.fr-fr.wxl
index 5f89947..374cd86 100644
--- a/build/windows/msi/i18n/scribe.fr-fr.wxl
+++ b/build/windows/msi/i18n/scribe.fr-fr.wxl
@@ -2,9 +2,9 @@
 <WixLocalization Culture="fr-fr" Codepage="1252" xmlns="http://schemas.microsoft.com/wix/2006/localization">
   <String Id="ProductLanguage">1036</String>
   <String Id="ProductName">@@PRODUCT_NAME@@</String>
-  <String Id="ProductHelpLink">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlInfoAbout">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlUpdateInfo">https://github.com/VSCodium/vscodium</String>
+  <String Id="ProductHelpLink">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlInfoAbout">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlUpdateInfo">https://github.com/BiblioNexus-Foundation/scribe</String>
   <String Id="PackageDescription">Package with @@PRODUCT_NAME@@</String>
   <String Id="FeatureStartMenuShortcutTitle">Start Menu shortcut</String>
   <String Id="FeatureStartMenuShortcutDescription">Create a start menu icon.</String>
diff --git a/build/windows/msi/i18n/scribe.it-it.wxl b/build/windows/msi/i18n/scribe.it-it.wxl
index f1b9f29..b88bbb9 100644
--- a/build/windows/msi/i18n/scribe.it-it.wxl
+++ b/build/windows/msi/i18n/scribe.it-it.wxl
@@ -2,9 +2,9 @@
 <WixLocalization Culture="it-it" Codepage="1252" xmlns="http://schemas.microsoft.com/wix/2006/localization">
   <String Id="ProductLanguage">1040</String>
   <String Id="ProductName">@@PRODUCT_NAME@@</String>
-  <String Id="ProductHelpLink">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlInfoAbout">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlUpdateInfo">https://github.com/VSCodium/vscodium</String>
+  <String Id="ProductHelpLink">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlInfoAbout">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlUpdateInfo">https://github.com/BiblioNexus-Foundation/scribe</String>
   <String Id="PackageDescription">Package with @@PRODUCT_NAME@@</String>
   <String Id="FeatureStartMenuShortcutTitle">Start Menu shortcut</String>
   <String Id="FeatureStartMenuShortcutDescription">Create a start menu icon.</String>
diff --git a/build/windows/msi/i18n/scribe.ja-jp.wxl b/build/windows/msi/i18n/scribe.ja-jp.wxl
index 42175b9..3d44c3c 100644
--- a/build/windows/msi/i18n/scribe.ja-jp.wxl
+++ b/build/windows/msi/i18n/scribe.ja-jp.wxl
@@ -2,9 +2,9 @@
 <WixLocalization Culture="ja-jp" Codepage="932" xmlns="http://schemas.microsoft.com/wix/2006/localization">
   <String Id="ProductLanguage">1041</String>
   <String Id="ProductName">@@PRODUCT_NAME@@</String>
-  <String Id="ProductHelpLink">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlInfoAbout">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlUpdateInfo">https://github.com/VSCodium/vscodium</String>
+  <String Id="ProductHelpLink">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlInfoAbout">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlUpdateInfo">https://github.com/BiblioNexus-Foundation/scribe</String>
   <String Id="PackageDescription">Package with @@PRODUCT_NAME@@</String>
   <String Id="FeatureStartMenuShortcutTitle">Start Menu shortcut</String>
   <String Id="FeatureStartMenuShortcutDescription">Create a start menu icon.</String>
diff --git a/build/windows/msi/i18n/scribe.ko-kr.wxl b/build/windows/msi/i18n/scribe.ko-kr.wxl
index d665d0b..9efdba4 100644
--- a/build/windows/msi/i18n/scribe.ko-kr.wxl
+++ b/build/windows/msi/i18n/scribe.ko-kr.wxl
@@ -2,9 +2,9 @@
 <WixLocalization Culture="ko-kr" Codepage="949" xmlns="http://schemas.microsoft.com/wix/2006/localization">
   <String Id="ProductLanguage">1042</String>
   <String Id="ProductName">@@PRODUCT_NAME@@</String>
-  <String Id="ProductHelpLink">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlInfoAbout">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlUpdateInfo">https://github.com/VSCodium/vscodium</String>
+  <String Id="ProductHelpLink">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlInfoAbout">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlUpdateInfo">https://github.com/BiblioNexus-Foundation/scribe</String>
   <String Id="PackageDescription">Package with @@PRODUCT_NAME@@</String>
   <String Id="FeatureStartMenuShortcutTitle">Start Menu shortcut</String>
   <String Id="FeatureStartMenuShortcutDescription">Create a start menu icon.</String>
diff --git a/build/windows/msi/i18n/scribe.ru-ru.wxl b/build/windows/msi/i18n/scribe.ru-ru.wxl
index e31d2dd..04bf128 100644
--- a/build/windows/msi/i18n/scribe.ru-ru.wxl
+++ b/build/windows/msi/i18n/scribe.ru-ru.wxl
@@ -2,9 +2,9 @@
 <WixLocalization Culture="ru-ru" Codepage="1251" xmlns="http://schemas.microsoft.com/wix/2006/localization">
   <String Id="ProductLanguage">1049</String>
   <String Id="ProductName">@@PRODUCT_NAME@@</String>
-  <String Id="ProductHelpLink">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlInfoAbout">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlUpdateInfo">https://github.com/VSCodium/vscodium</String>
+  <String Id="ProductHelpLink">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlInfoAbout">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlUpdateInfo">https://github.com/BiblioNexus-Foundation/scribe</String>
   <String Id="PackageDescription">Пакет с @@PRODUCT_NAME@@</String>
   <String Id="FeatureStartMenuShortcutTitle">Ярлык в меню «Пуск»</String>
   <String Id="FeatureStartMenuShortcutDescription">Создайте значок в меню «Пуск».</String>
diff --git a/build/windows/msi/i18n/scribe.zh-cn.wxl b/build/windows/msi/i18n/scribe.zh-cn.wxl
index c5fab3a..ff3605d 100644
--- a/build/windows/msi/i18n/scribe.zh-cn.wxl
+++ b/build/windows/msi/i18n/scribe.zh-cn.wxl
@@ -2,9 +2,9 @@
 <WixLocalization Culture="zh-cn" Codepage="936" xmlns="http://schemas.microsoft.com/wix/2006/localization">
   <String Id="ProductLanguage">2052</String>
   <String Id="ProductName">@@PRODUCT_NAME@@</String>
-  <String Id="ProductHelpLink">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlInfoAbout">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlUpdateInfo">https://github.com/VSCodium/vscodium</String>
+  <String Id="ProductHelpLink">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlInfoAbout">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlUpdateInfo">https://github.com/BiblioNexus-Foundation/scribe</String>
   <String Id="PackageDescription">Package with @@PRODUCT_NAME@@</String>
   <String Id="FeatureStartMenuShortcutTitle">Start Menu shortcut</String>
   <String Id="FeatureStartMenuShortcutDescription">Create a start menu icon.</String>
diff --git a/build/windows/msi/i18n/scribe.zh-tw.wxl b/build/windows/msi/i18n/scribe.zh-tw.wxl
index a218f51..645e016 100644
--- a/build/windows/msi/i18n/scribe.zh-tw.wxl
+++ b/build/windows/msi/i18n/scribe.zh-tw.wxl
@@ -2,9 +2,9 @@
 <WixLocalization Culture="zh-tw" Codepage="950" xmlns="http://schemas.microsoft.com/wix/2006/localization">
   <String Id="ProductLanguage">1028</String>
   <String Id="ProductName">@@PRODUCT_NAME@@</String>
-  <String Id="ProductHelpLink">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlInfoAbout">https://github.com/VSCodium/vscodium</String>
-  <String Id="ProductUrlUpdateInfo">https://github.com/VSCodium/vscodium</String>
+  <String Id="ProductHelpLink">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlInfoAbout">https://github.com/BiblioNexus-Foundation/scribe</String>
+  <String Id="ProductUrlUpdateInfo">https://github.com/BiblioNexus-Foundation/scribe</String>
   <String Id="PackageDescription">Package with @@PRODUCT_NAME@@</String>
   <String Id="FeatureStartMenuShortcutTitle">Start Menu shortcut</String>
   <String Id="FeatureStartMenuShortcutDescription">Create a start menu icon.</String>
diff --git a/build/windows/msi/scribe.wxs b/build/windows/msi/scribe.wxs
index 3734d65..0fc6687 100644
--- a/build/windows/msi/scribe.wxs
+++ b/build/windows/msi/scribe.wxs
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension" xmlns:netfx="http://schemas.microsoft.com/wix/NetFxExtension">
-  <?include $(sys.CURRENTDIR)includes\vscodium-variables.wxi ?>
+  <?include $(sys.CURRENTDIR)includes\scribe-variables.wxi ?>
 
   <Product Id="$(var.ProductId)" Name="$(var.ProductNameWithVersion)" Language="$(var.ProductLanguage)" Version="$(var.ProductVersion)" Manufacturer="$(var.ProductManufacturerLong)" UpgradeCode="$(var.ProductUpgradeCode)">
     <Package Description="!(loc.PackageDescription)" Manufacturer="$(var.ProductManufacturerLong)" InstallerVersion="200" Compressed="yes" />
@@ -13,7 +13,7 @@
     <SetProperty Id="ARPINSTALLLOCATION" Value="[APPLICATIONFOLDER]" After="CostFinalize" />
     <Icon Id="code.ico" SourceFile="$(var.IconDir)\code.ico" />
 
-    <!-- Microsoft .NET Framework 4.5.2 is required for VSCodium. If you are using Windows 7, please make sure .NET Framework 4.5.2 is installed. -->
+    <!-- Microsoft .NET Framework 4.5.2 is required for Scribe. If you are using Windows 7, please make sure .NET Framework 4.5.2 is installed. -->
     <PropertyRef Id='WIX_IS_NETFRAMEWORK_452_OR_LATER_INSTALLED' />
 	<Condition Message="!(loc.MinimumNetFramworkRequired)">
       <![CDATA[Installed OR WIX_IS_NETFRAMEWORK_452_OR_LATER_INSTALLED]]>
@@ -39,7 +39,7 @@
       <Custom Action="PreventDowngrading" After="FindRelatedProducts">NEWPRODUCTFOUND</Custom>
       <!--
         In general it seems very hard to track what files may have changed
-        between vscodium versions. As files are crawled with heat and GUIDs
+        between scribe versions. As files are crawled with heat and GUIDs
         are automatically generated some MSI "optimization" features are not
         available. But no real problem, it only takes some more seconds to
         upgrade between versions.
@@ -81,7 +81,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="ascx" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).ascx" Description="ASCX" Icon="XML.ICO">
               <Extension Id="ascx">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -89,7 +89,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="asp" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).asp" Description="ASP" Icon="HTML.ICO">
               <Extension Id="asp">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -97,7 +97,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="aspx" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).aspx" Description="ASPX" Icon="HTML.ICO">
               <Extension Id="aspx">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -105,7 +105,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="bash" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).bash" Description="Bash" Icon="SHELL.ICO">
               <Extension Id="bash">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -113,7 +113,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="bash_login" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).bash_login" Description="Bash Login" Icon="SHELL.ICO">
               <Extension Id="bash_login">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -121,7 +121,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="bash_logout" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).bash_logout" Description="Bash Logout" Icon="SHELL.ICO">
               <Extension Id="bash_logout">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -129,7 +129,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="bash_profile" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).bash_profile" Description="Bash Profile" Icon="SHELL.ICO">
               <Extension Id="bash_profile">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -137,7 +137,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="bashrc" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).bashrc" Description="Bash RC" Icon="SHELL.ICO">
               <Extension Id="bashrc">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -145,7 +145,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="bib" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).bib" Description="BibTeX" Icon="DEFAULT.ICO">
               <Extension Id="bib">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -153,7 +153,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="bowerrc" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).bowerrc" Description="Bower RC" Icon="BOWER.ICO">
               <Extension Id="bowerrc">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -161,7 +161,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="c" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).c" Description="C" Icon="C.ICO">
               <Extension Id="c">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -169,7 +169,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="cc" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).cc" Description="C++" Icon="CPP.ICO">
               <Extension Id="cc">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -177,7 +177,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="clj" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).clj" Description="Clojure" Icon="DEFAULT.ICO">
               <Extension Id="clj">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -185,7 +185,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="cljs" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).cljs" Description="ClojureScript" Icon="DEFAULT.ICO">
               <Extension Id="cljs">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -193,7 +193,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="cljx" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).cljx" Description="CLJX" Icon="DEFAULT.ICO">
               <Extension Id="cljx">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -201,7 +201,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="clojure" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).clojure" Description="Clojure" Icon="DEFAULT.ICO">
               <Extension Id="clojure">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -209,7 +209,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="code-workspace" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).code-workspace" Description="Code Workspace" Icon="DEFAULT.ICO">
               <Extension Id="code-workspace">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -217,7 +217,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="coffee" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).coffee" Description="CoffeeScript" Icon="DEFAULT.ICO">
               <Extension Id="coffee">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -225,7 +225,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="config" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).config" Description="Configuration" Icon="CONFIG.ICO">
               <Extension Id="config">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -233,7 +233,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="cpp" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).cpp" Description="C++" Icon="CPP.ICO">
               <Extension Id="cpp">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -241,7 +241,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="cs" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).cs" Description="C#" Icon="CSHARP.ICO">
               <Extension Id="cs">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -249,7 +249,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="cshtml" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).cshtml" Description="CSHTML" Icon="HTML.ICO">
               <Extension Id="cshtml">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -257,7 +257,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="csproj" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).csproj" Description="C# Project" Icon="XML.ICO">
               <Extension Id="csproj">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -265,7 +265,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="css" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).css" Description="CSS" Icon="CSS.ICO">
               <Extension Id="css">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -273,7 +273,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="csx" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).csx" Description="C# Script" Icon="CSHARP.ICO">
               <Extension Id="csx">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -281,7 +281,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="ctp" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).ctp" Description="CakePHP Template" Icon="DEFAULT.ICO">
               <Extension Id="ctp">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -289,7 +289,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="cxx" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).cxx" Description="C++" Icon="CPP.ICO">
               <Extension Id="cxx">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -297,7 +297,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="dockerfile" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).dockerfile" Description="Dockerfile" Icon="DEFAULT.ICO">
               <Extension Id="dockerfile">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -305,7 +305,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="dot" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).dot" Description="Dot" Icon="DEFAULT.ICO">
               <Extension Id="dot">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -313,7 +313,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="dtd" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).dtd" Description="Document Type Definition" Icon="XML.ICO">
               <Extension Id="dtd">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -321,7 +321,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="editorconfig" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).editorconfig" Description="Editor Config" Icon="CONFIG.ICO">
               <Extension Id="editorconfig">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -329,7 +329,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="edn" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).edn" Description="Extensible Data Notation" Icon="DEFAULT.ICO">
               <Extension Id="edn">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -337,7 +337,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="eyaml" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).eyaml" Description="Hiera Eyaml" Icon="YAML.ICO">
               <Extension Id="eyaml">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -345,7 +345,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="eyml" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).eyml" Description="Hiera Eyaml" Icon="YAML.ICO">
               <Extension Id="eyml">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -353,7 +353,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="fs" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).fs" Description="F#" Icon="DEFAULT.ICO">
               <Extension Id="fs">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -361,7 +361,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="fsi" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).fsi" Description="F# Signature" Icon="DEFAULT.ICO">
               <Extension Id="fsi">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -369,7 +369,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="fsscript" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).fsscript" Description="F# Script" Icon="DEFAULT.ICO">
               <Extension Id="fsscript">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -377,7 +377,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="fsx" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).fsx" Description="F# Script" Icon="DEFAULT.ICO">
               <Extension Id="fsx">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -385,7 +385,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="gemspec" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).gemspec" Description="Gemspec" Icon="RUBY.ICO">
               <Extension Id="gemspec">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -393,7 +393,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="gitattributes" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).gitattributes" Description="Git Attributes" Icon="CONFIG.ICO">
               <Extension Id="gitattributes">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -401,7 +401,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="gitconfig" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).gitconfig" Description="Git Config" Icon="CONFIG.ICO">
               <Extension Id="gitconfig">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -409,7 +409,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="gitignore" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).gitignore" Description="Git Ignore" Icon="CONFIG.ICO">
               <Extension Id="gitignore">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -417,7 +417,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="go" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).go" Description="Go" Icon="GO.ICO">
               <Extension Id="go">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -425,7 +425,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="h" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).h" Description="C Header" Icon="C.ICO">
               <Extension Id="h">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -433,7 +433,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="handlebars" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).handlebars" Description="Handlebars" Icon="DEFAULT.ICO">
               <Extension Id="handlebars">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -441,7 +441,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="hbs" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).hbs" Description="Handlebars" Icon="DEFAULT.ICO">
               <Extension Id="hbs">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -449,7 +449,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="hh" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).hh" Description="C++ Header" Icon="CPP.ICO">
               <Extension Id="hh">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -457,7 +457,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="hpp" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).hpp" Description="C++ Header" Icon="CPP.ICO">
               <Extension Id="hpp">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -465,7 +465,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="htm" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).htm" Description="HTML" Icon="HTML.ICO">
               <Extension Id="htm">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -473,7 +473,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="html" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).html" Description="HTML" Icon="HTML.ICO">
               <Extension Id="html">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -481,7 +481,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="hxx" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).hxx" Description="C++ Header" Icon="CPP.ICO">
               <Extension Id="hxx">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -489,7 +489,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="ini" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).ini" Description="INI" Icon="CONFIG.ICO">
               <Extension Id="ini">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -497,7 +497,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="jade" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).jade" Description="Jade" Icon="JADE.ICO">
               <Extension Id="jade">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -505,7 +505,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="jav" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).jav" Description="Java" Icon="JAVA.ICO">
               <Extension Id="jav">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -513,7 +513,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="java" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).java" Description="Java" Icon="JAVA.ICO">
               <Extension Id="java">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -521,7 +521,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="js" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).js" Description="JavaScript" Icon="JAVASCRIPT.ICO">
               <Extension Id="js">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -529,7 +529,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="jsx" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).jsx" Description="JavaScript" Icon="REACT.ICO">
               <Extension Id="jsx">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -537,7 +537,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="jscsrc" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).jscsrc" Description="JSCS RC" Icon="JAVASCRIPT.ICO">
               <Extension Id="jscsrc">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -545,7 +545,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="jshintrc" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).jshintrc" Description="JSHint RC" Icon="JAVASCRIPT.ICO">
               <Extension Id="jshintrc">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -553,7 +553,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="jshtm" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).jshtm" Description="JavaScript HTML Template" Icon="HTML.ICO">
               <Extension Id="jshtm">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -561,7 +561,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="json" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).json" Description="JSON" Icon="JSON.ICO">
               <Extension Id="json">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -569,7 +569,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="jsp" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).jsp" Description="Java Server Pages" Icon="HTML.ICO">
               <Extension Id="jsp">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -577,7 +577,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="less" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).less" Description="LESS" Icon="LESS.ICO">
               <Extension Id="less">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -585,7 +585,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="lua" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).lua" Description="Lua" Icon="DEFAULT.ICO">
               <Extension Id="lua">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -593,7 +593,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="m" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).m" Description="Objective C" Icon="DEFAULT.ICO">
               <Extension Id="m">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -601,7 +601,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="makefile" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).makefile" Description="Makefile" Icon="DEFAULT.ICO">
               <Extension Id="makefile">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -609,7 +609,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="markdown" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).markdown" Description="Markdown" Icon="MARKDOWN.ICO">
               <Extension Id="markdown">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -617,7 +617,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="md" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).md" Description="Markdown" Icon="MARKDOWN.ICO">
               <Extension Id="md">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -625,7 +625,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="mdoc" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).mdoc" Description="MDoc" Icon="MARKDOWN.ICO">
               <Extension Id="mdoc">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -633,7 +633,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="mdown" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).mdown" Description="Markdown" Icon="MARKDOWN.ICO">
               <Extension Id="mdown">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -641,7 +641,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="mdtext" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).mdtext" Description="Markdown" Icon="MARKDOWN.ICO">
               <Extension Id="mdtext">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -649,7 +649,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="mdtxt" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).mdtxt" Description="Markdown" Icon="MARKDOWN.ICO">
               <Extension Id="mdtxt">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -657,7 +657,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="mdwn" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).mdwn" Description="Markdown" Icon="MARKDOWN.ICO">
               <Extension Id="mdwn">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -665,7 +665,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="mkd" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).mkd" Description="Markdown" Icon="MARKDOWN.ICO">
               <Extension Id="mkd">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -673,7 +673,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="mkdn" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).mkdn" Description="Markdown" Icon="MARKDOWN.ICO">
               <Extension Id="mkdn">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -681,7 +681,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="ml" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).ml" Description="OCaml" Icon="DEFAULT.ICO">
               <Extension Id="ml">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -689,7 +689,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="mli" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).mli" Description="OCaml" Icon="DEFAULT.ICO">
               <Extension Id="mli">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -697,7 +697,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="mjs" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).mjs" Description="JavaScript" Icon="JAVASCRIPT.ICO">
               <Extension Id="mjs">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -705,7 +705,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="npmignore" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).npmignore" Description="NPM Ignore" Icon="DEFAULT.ICO">
               <Extension Id="npmignore">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -713,7 +713,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="php" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).php" Description="PHP" Icon="DEFAULT.ICO">
               <Extension Id="php">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -721,7 +721,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="phtml" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).phtml" Description="PHP HTML" Icon="PHP.ICO">
               <Extension Id="phtml">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -729,7 +729,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="pl" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).pl" Description="Perl" Icon="DEFAULT.ICO">
               <Extension Id="pl">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -737,7 +737,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="pl6" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).pl6" Description="Perl 6" Icon="DEFAULT.ICO">
               <Extension Id="pl6">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -745,7 +745,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="pm" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).pm" Description="Perl Module" Icon="DEFAULT.ICO">
               <Extension Id="pm">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -753,7 +753,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="pm6" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).pm6" Description="Perl 6 Module" Icon="DEFAULT.ICO">
               <Extension Id="pm6">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -761,7 +761,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="pod" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).pod" Description="Perl POD" Icon="DEFAULT.ICO">
               <Extension Id="pod">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -769,7 +769,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="pp" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).pp" Description="Perl" Icon="DEFAULT.ICO">
               <Extension Id="pp">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -777,7 +777,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="profile" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).profile" Description="Profile" Icon="SHELL.ICO">
               <Extension Id="profile">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -785,7 +785,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="properties" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).properties" Description="Properties" Icon="DEFAULT.ICO">
               <Extension Id="properties">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -793,7 +793,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="ps1" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).ps1" Description="PowerShell" Icon="POWERSHELL.ICO">
               <Extension Id="ps1">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -801,7 +801,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="psd1" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).psd1" Description="PowerShell" Icon="POWERSHELL.ICO">
               <Extension Id="psd1">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -809,7 +809,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="psgi" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).psgi" Description="Perl CGI" Icon="DEFAULT.ICO">
               <Extension Id="psgi">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -817,7 +817,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="psm1" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).psm1" Description="PowerShell Module" Icon="POWERSHELL.ICO">
               <Extension Id="psm1">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -825,7 +825,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="py" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).py" Description="Python" Icon="PYTHON.ICO">
               <Extension Id="py">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -833,7 +833,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="r" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).r" Description="R" Icon="DEFAULT.ICO">
               <Extension Id="r">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -841,7 +841,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="rb" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).rb" Description="Ruby" Icon="RUBY.ICO">
               <Extension Id="rb">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -849,7 +849,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="rhistory" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).rhistory" Description="R History" Icon="SHELL.ICO">
               <Extension Id="rhistory">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -857,7 +857,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="rhistory" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).rprofile" Description="R Profile" Icon="SHELL.ICO">
               <Extension Id="rprofile">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -865,7 +865,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="rs" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).rs" Description="Rust" Icon="DEFAULT.ICO">
               <Extension Id="rs">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -873,7 +873,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="rt" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).rt" Description="Rich Text" Icon="DEFAULT.ICO">
               <Extension Id="rt">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -881,7 +881,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="scss" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).scss" Description="Sass" Icon="SASS.ICO">
               <Extension Id="scss">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -889,7 +889,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="sh" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).sh" Description="SH" Icon="SHELL.ICO">
               <Extension Id="sh">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -897,7 +897,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="shtml" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).shtml" Description="SHTML" Icon="HTML.ICO">
               <Extension Id="shtml">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -905,7 +905,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="sql" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).sql" Description="SQL" Icon="SQL.ICO">
               <Extension Id="sql">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -913,7 +913,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="svg" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).svg" Description="SVG" Icon="DEFAULT.ICO">
               <Extension Id="svg">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -921,7 +921,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="svgz" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).svgz" Description="SVGZ" Icon="DEFAULT.ICO">
               <Extension Id="svgz">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -929,7 +929,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="t" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).t" Description="Perl" Icon="DEFAULT.ICO">
               <Extension Id="t">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -937,7 +937,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="tex" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).tex" Description="LaTeX" Icon="DEFAULT.ICO">
               <Extension Id="tex">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -945,7 +945,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="ts" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).ts" Description="TypeScript" Icon="TYPESCRIPT.ICO">
               <Extension Id="ts">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -953,7 +953,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="tsx" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).tsx" Description="TypeScript" Icon="REACT.ICO">
               <Extension Id="tsx">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -961,7 +961,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="txt" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).txt" Description="Text" Icon="DEFAULT.ICO">
               <Extension Id="txt">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -969,7 +969,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="vb" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).vb" Description="Visual Basic" Icon="DEFAULT.ICO">
               <Extension Id="vb">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -977,7 +977,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="vue" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).vue" Description="VUE" Icon="VUE.ICO">
               <Extension Id="vue">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -985,7 +985,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="wxi" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).wxi" Description="WiX Include" Icon="DEFAULT.ICO">
               <Extension Id="wxi">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -993,7 +993,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="wxl" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).wxl" Description="WiX Localization" Icon="DEFAULT.ICO">
               <Extension Id="wxl">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -1001,7 +1001,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="wxs" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).wxs" Description="WiX" Icon="DEFAULT.ICO">
               <Extension Id="wxs">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -1009,7 +1009,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="xaml" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).xaml" Description="XAML" Icon="XML.ICO">
               <Extension Id="xaml">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -1017,7 +1017,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="xml" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).xml" Description="XML" Icon="XML.ICO">
               <Extension Id="xml">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -1025,7 +1025,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="yaml" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).yaml" Description="Yaml" Icon="YAML.ICO">
               <Extension Id="yaml">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -1033,7 +1033,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="yml" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).yml" Description="Yaml" Icon="YAML.ICO">
               <Extension Id="yml">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -1041,7 +1041,7 @@
             <RegistryValue Root="HKLM" Key="SOFTWARE\$(var.ProductManufacturerShort)\$(var.AppName)\Components\FileTypeAssociations" Name="zsh" Type="integer" Value="1" />
             <ProgId Id="$(var.AppCodeName).zsh" Description="ZSH" Icon="SHELL.ICO">
               <Extension Id="zsh">
-                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="VSCODIUM.EXE" />
+                <Verb Id="open" Command="&amp;Open" Argument="&quot;%1&quot;" TargetFile="SCRIBE.EXE" />
               </Extension>
             </ProgId>
           </Component>
@@ -1501,7 +1501,7 @@
 
     <Property Id="WIXUI_EXITDIALOGOPTIONALCHECKBOX" Value="1" />
     <Property Id="WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT" Value="!(loc.LaunchApplication)" />
-    <Property Id="WixShellExecTarget" Value="[#VSCODIUM.EXE]" />
+    <Property Id="WixShellExecTarget" Value="[#SCRIBE.EXE]" />
     <CustomAction Id="LaunchApplication" BinaryKey="WixCA" DllEntry="WixShellExec" Impersonate="yes" />
   </Product>
 </Wix>
diff --git a/build/windows/msi/scribe.xsl b/build/windows/msi/scribe.xsl
index 87ec399..5ffa1bc 100644
--- a/build/windows/msi/scribe.xsl
+++ b/build/windows/msi/scribe.xsl
@@ -14,14 +14,14 @@
   <xsl:key name="vId1ToReplace" match="wi:Component[wi:File[contains(@Source,'@@PRODUCT_NAME@@.exe')]]" use="@Id"/>
   <xsl:template match="node()[key('vId1ToReplace', @Id)]">
     <xsl:copy>
-      <xsl:attribute name="Id">VSCODIUM.EXE</xsl:attribute>
+      <xsl:attribute name="Id">SCRIBE.EXE</xsl:attribute>
       <xsl:copy-of select="@*[name()!='Id']"/>
       <xsl:apply-templates />
     </xsl:copy>
   </xsl:template>
   <xsl:template match="wi:Component/wi:File[contains(@Source,'@@PRODUCT_NAME@@.exe')]">
      <xsl:copy>
-        <xsl:attribute name="Id">VSCODIUM.EXE</xsl:attribute>
+        <xsl:attribute name="Id">SCRIBE.EXE</xsl:attribute>
         <xsl:copy-of select="@*[name()!='Id']"/>
         <xsl:apply-templates />
      </xsl:copy>
